packages/core/src/extensions/Blocks/nodes/Block.ts

```diff
@@ -506,49 +536,49 @@ export const Block = Node.create<IBlock>({
         this.editor.commands.BNSetContentType(
           this.editor.state.selection.anchor,
           {
-            name: "headingContent",
+            name: "heading",
             attrs: {
-              headingLevel: "1",
+              level: "1",
             },
           }
         ),
       "Mod-Alt-2": () =>
         this.editor.commands.BNSetContentType(
           this.editor.state.selection.anchor,
           {
-            name: "headingContent",
+            name: "heading",
             attrs: {
-              headingLevel: "2",
+              level: "2",
             },
           }
         ),
       "Mod-Alt-3": () =>
         this.editor.commands.BNSetContentType(
           this.editor.state.selection.anchor,
           {
-            name: "headingContent",
+            name: "heading",
             attrs: {
-              headingLevel: "3",
+              level: "3",
             },
           }
         ),
       "Mod-Shift-7": () =>
         this.editor.commands.BNSetContentType(
           this.editor.state.selection.anchor,
           {
-            name: "listItemContent",
+            name: "listItem",
             attrs: {
-              listItemType: "unordered",
+              ordered: "false",
```

Comment:
Is there a specific reason you went from "type" to "ordered"?

I think "type" makes more sense, as we might also have a "check" type for a check (todo) list, etc.

Reply:
The reason is because the naming clashes with the type parameter, and also because I think there could be a lot of attributes for other block types in the future which would also want to use a type attribute, so imo it's good to have it be called something unique to list items. Though the checked lists are a good point. Maybe `listType` wouldn't be bad?